pom.xml

```diff
@@ -1,13 +1,20 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+
     <groupId>com.salesforce.conduit</groupId>
     <artifactId>emp-connector</artifactId>
     <version>0.0.1-SNAPSHOT</version>
     <name>EMP Connector</name>
     <description>A simplified connector for EMP</description>
     <url>https://github.com/forcedotcom/EMP-Connector</url>
```

Comment:
I think this "url" tag is just for documentation, but you might remove it or change it to the mainstreethub fork. Probably not a big deal, though.